test/Passes/pass-hoist-vector-transfer-operation-brgemm.mlir

```diff
@@ -0,0 +1,382 @@
+// RUN: tpp-opt %s  --hoist-vector-transfer --split-input-file  | FileCheck %s
+
+
+#map = affine_map<(d0, d1, d2, d3) -> (d0, d1, d3)>
+#map1 = affine_map<(d0, d1, d2, d3) -> (d0, d3, d2)>
+#map2 = affine_map<(d0, d1, d2, d3) -> (d1, d2)>
+module {
+  memref.global "private" constant @__constant_24x64x64xf32 : memref<24x64x64xf32> = dense<1.000000e+00> {alignment = 64 : i64}
+  func.func @entry(%arg0: memref<8x24x32x64xf32>) -> memref<8x24x32x64xf32> {
```

Comment:
please name the functions as to what they're doing. It's hard to see the difference between the multiple entries in this test

Reply:
Yes, Named the functions.